.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Ragna documentation
+    url: https://docs.ragna.chat/
+    about: Check out the Ragna documentation
+  - name: Blank issue (maintainers only)
```

Comment:
Do L1 and L6 conflict?

Reply:
No, it's an alternative chosen to be able to add addition: "maintainers only" text there

Reply:
@pavithraes Are you sure? The contact links do not show up:

![image](https://github.com/Quansight/ragna/assets/6849766/4b5be39a-3df4-44fe-9c6d-4e858f756fc3)

Reply:
I was wrong - we do need to omit the first line. Sorry!